mycroft/skills/intent_service.py

```diff
@@ -531,30 +336,33 @@ def handle_remove_context(self, message):
         """
         context = message.data.get('context')
         if context:
-            self.context_manager.remove_context(context)
+            self.adapt_service.context_manager.remove_context(context)
 
     def handle_clear_context(self, message):
         """Clears all keywords from context """
-        self.context_manager.clear_context()
+        self.adapt_service.context_manager.clear_context()
 
     def handle_get_adapt(self, message):
         utterance = message.data["utterance"]
         lang = message.data.get("lang", "en-us")
         norm = normalize(utterance, lang, remove_articles=False)
-        intent = self._adapt_intent_match([utterance], [norm], lang)
+        combined = [utterance, norm] if utterance != norm else [utterance]
+        intent = self.adapt_service.match_intent(utterance, lang)
```

Comment:
I missed this one - mismatch between `combined` and `utterance`

Reply:
Fixed